recipes/metview/build.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+set -e # Abort on error.
+
+export PING_SLEEP=30s
+export WORKDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
It's more clear to give preference to variables from `conda build`: https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html (`RECIPE_DIR` in this case)

Reply:
Done